### PR TITLE
Restored AC-8362 - Reflect more inclusive options for gender

### PR DESCRIPTION
**Pull request by pbx**
## [AC-8362](https://masschallenge.atlassian.net/browse/AC-8362)

_[Original PR](https://github.com/masschallenge/django-accelerator/pull/362)_

### CODE REVIEW

**Comment by iankigen** [comment]
Though not specified as part of def of done on this ticket, the help_text: **"Select as many options as you feel best represent your identity. Press and hold Control on PCs or Command on Macs and click to select multiple options"** appear to be part of the requirement according to the [mock-up](https://www.figma.com/proto/8zEeyAcxQCFtNf1oQtuAsF/Demographic-Information?node-id=163%3A1&viewport=522%2C250%2C0.22464416921138763&scaling=min-zoom) on dependent tickets. @hfreund should we include the help text as part of this work?
Screenshot 2021-01-15 at 14 11 25

**Comment by hfreund** [comment]
> Though not specified as part of def of done on this ticket, the help_text: **"Select as many options as you feel best represent your identity. Press and hold Control on PCs or Command on Macs and click to select multiple options"** appear to be part of the requirement according to the [mock-up](https://www.figma.com/proto/8zEeyAcxQCFtNf1oQtuAsF/Demographic-Information?node-id=163%3A1&viewport=522%2C250%2C0.22464416921138763&scaling=min-zoom) on dependent tickets. @hfreund should we include the help text as part of this work?

Good catch, Ian. I'm not sure if this PR is the right place to make code changes - is it possible that might over-complicate an already complicated process? My feeling is it would be perhaps easier to make a small follow-on ticket to add the help text.

**Review by PrinceDavis — APPROVED** [review]
gender field available ✅ 
migration ran ✅

**Review by philophilo — APPROVED** [review]
✅ migration sequencing looks great 
✅ migrations run to completion

File: accelerator/migrations/0028_add_gender_choices_object.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                ('updated_at', models.DateTimeField(auto_now=True,
+                                                    null=True)),
+                ('name', models.CharField(max_length=255, unique=True)),
+            ],
+            options={
+                'verbose_name_plural': 'Gender Choices',
+                'db_table': 'accelerator_genderchoices',
+                'abstract': False,
+                'managed': True,
+                'swappable': 'ACCELERATOR_GENDERCHOICES_MODEL',
```
Comment (iankigen) [inline accelerator/migrations/0028_add_gender_choices_object.py:31]: To avoid migration issues on this model going forward, this needs to be set to None

Comment (PrinceDavis) [reply accelerator/migrations/0028_add_gender_choices_object.py:31]: @iankigen are you requesting for change here?

Comment (philophilo) [reply accelerator/migrations/0028_add_gender_choices_object.py:31]: yes, you are right, missed that

Comment (iankigen) [reply accelerator/migrations/0028_add_gender_choices_object.py:31]: yeah, it seems I was 15 seconds late

Comment (iankigen) [reply accelerator/migrations/0028_add_gender_choices_object.py:31]: can we revert and update the swappable option to None?

**Pull request merged**